src/library/dlgtagfetcher.cpp

```diff
@@ -222,6 +234,29 @@ void DlgTagFetcher::apply() {
             QDateTime::currentDateTimeUtc());
 }
 
+void DlgTagFetcher::retry() {
+    results->clear();
+    disconnect(m_track.get(),
+            &Track::changed,
+            this,
+            &DlgTagFetcher::slotTrackChanged);
+
+    addDivider(tr("Original tags"), results);
+    addTrack(trackColumnValues(*m_track), -1, results);
```

Comment:
Can we remove this code or find a workaround that this can be removed?

Reply:
done 👍